frontend/pages/_document.tsx

```diff
@@ -7,10 +7,6 @@ export default function Document() {
         <base href="/" />
 
         <link rel="icon" type="image/png" href="./favicon.png" />
-        <link
```

Comment:
Is this just a cleanup or changing functionality? Doing it like this should let font optimization work https://nextjs.org/docs/basic-features/font-optimization which currently seems to be google fonts only, but hopefully more in the future.